composer.lock

```diff
@@ -0,0 +1,3251 @@
+{
```

Comment:
Ah, I guess I should not have committed the composer.lock.

Reply:
I think I have been able to remove this from history and add a .gitignore file in place, no worries